rapids-cmake/test/detail/generate_installed_CTestTestfile.cmake

```diff
@@ -19,6 +19,10 @@ cmake_policy(SET CMP0007 NEW) # allow empty list entries
 cmake_policy(SET CMP0009 NEW) # don't follow symlinks
 cmake_policy(SET CMP0057 NEW) # allow `if( IN_LIST )`
 
+if(POLICY CMP0159)
```

Comment:
Is there a reason we're not doing:

```cmake
cmake_minimum_required(VERSION 3.26...3.29)
```

?

Reply:
I mentally had thought that `cmake_minimum_required` wasn't supported in script mode. I will verify that this works, since ctest script mode is slightly different compared to cmake's.

Reply:
This works. Updated

Thanks @KyleFromNVIDIA